### PR TITLE
Feature/cw 82 title description and favicon

### PR DESCRIPTION
**Pull request by luisrca-tech**

### CODE REVIEW

File: public/favicon.svg
Comment (mateusmenesesDev) [inline public/favicon.svg:1]: Nós tem um favicon específico do attios crm. É o icone que fica no header

**Review by mateusmenesesDev — CHANGES_REQUESTED** [review]

File: public/favicon.svg
Comment (luisrca-tech) [reply public/favicon.svg:1]: Alterei pra ele, da uma olhada pra ver se você gosta ou se fica um pouco estranho... pq ele é uma logo inteira.

Comment (mateusmenesesDev) [reply public/favicon.svg:1]: Group 11

Use esse

Comment (mateusmenesesDev) [reply public/favicon.svg:1]: <svg width="31" height="25" viewBox="0 0 31 25" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M22.3217 7.48047C22.7328 6.82077 22.7328 5.97237 22.3217 5.31647L19.8101 1.29689L19.6008 0.95865C19.2252 0.3588 18.5787 0 17.8685 0H12.1989C11.4907 0 10.8441 0.3588 10.4666 0.96052L0.313942 17.2089C0.110254 17.5341 0 17.9097 0 18.2909C0 18.6721 0.108385 19.0477 0.312074 19.371L3.03478 23.7307C3.41038 24.3324 4.05696 24.6893 4.76519 24.6893H10.4348C11.1468 24.6893 11.7934 24.3306 12.1671 23.7288L12.3746 23.3999C12.3746 23.3999 12.3746 23.3999 12.3746 23.3962L12.3783 23.3887L14.4021 20.1521L20.4006 10.5526L22.3179 7.48227L22.3217 7.48047ZM21.7293 6.39847C21.7293 6.60587 21.6714 6.81517 21.5536 7.00017L11.6084 22.9178C11.5187 23.0617 11.3748 23.0785 11.3169 23.0785C11.2589 23.0785 11.1169 23.0617 11.0254 22.9178L8.51195 18.8908C8.28396 18.5264 8.28396 18.0573 8.51195 17.6892L18.4572 1.77527C18.5469 1.62951 18.6908 1.6127 18.7487 1.6127C18.8066 1.6127 18.9505 1.62951 19.0421 1.77714L21.5536 5.79677C21.6714 5.98177 21.7293 6.19097 21.7293 6.39847Z" fill="#1D1F20"/>
<path d="M29.7538 17.2063L27.2422 13.1867C27.2422 13.1867 27.2329 13.1699 27.2273 13.1624L27.0292 12.8466C26.6555 12.2468 26.0089 11.888 25.3025 11.8861L21.2568 11.873L20.9746 12.3253L16.1403 20.0617L15.873 20.4896L17.8987 23.7262C18.2725 24.328 18.919 24.6867 19.631 24.6867H25.3007C25.9996 24.6867 26.6611 24.3186 27.0311 23.7281L27.231 23.4086C27.231 23.4086 27.2385 23.3992 27.2404 23.3955L29.7556 19.3703C30.1686 18.7125 30.1686 17.8622 29.7556 17.2063H29.7538ZM28.9876 18.89L26.4723 22.9152C26.4611 22.9339 26.448 22.9488 26.4368 22.9638C26.349 23.0628 26.235 23.0759 26.1846 23.0759C26.1266 23.0759 25.9846 23.0591 25.893 22.9133L23.3778 18.8882C23.3497 18.8433 23.3254 18.7966 23.303 18.7461C23.2806 18.6975 23.2638 18.649 23.2488 18.5985C23.1928 18.3967 23.1928 18.1799 23.2488 17.9781C23.2769 17.8791 23.3198 17.78 23.3759 17.6903L25.8874 13.6689C25.8874 13.6689 25.8912 13.6633 25.893 13.6595C25.9528 13.5698 26.0276 13.5287 26.093 13.5156C26.1192 13.5082 26.1416 13.5063 26.1603 13.5026C26.1696 13.5026 26.1789 13.5026 26.1883 13.5026C26.2462 13.5026 26.3901 13.5212 26.4798 13.667L28.9913 17.6866C29.2212 18.0528 29.2212 18.5238 28.9913 18.89H28.9876Z" fill="#1D1F20"/>
</svg>

Comment (mateusmenesesDev) [reply public/favicon.svg:1]: <svg width="31" height="25" viewBox="0 0 31 25" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M22.3217 7.48047C22.7328 6.82077 22.7328 5.97237 22.3217 5.31647L19.8101 1.29689L19.6008 0.95865C19.2252 0.3588 18.5787 0 17.8685 0H12.1989C11.4907 0 10.8441 0.3588 10.4666 0.96052L0.313942 17.2089C0.110254 17.5341 0 17.9097 0 18.2909C0 18.6721 0.108385 19.0477 0.312074 19.371L3.03478 23.7307C3.41038 24.3324 4.05696 24.6893 4.76519 24.6893H10.4348C11.1468 24.6893 11.7934 24.3306 12.1671 23.7288L12.3746 23.3999C12.3746 23.3999 12.3746 23.3999 12.3746 23.3962L12.3783 23.3887L14.4021 20.1521L20.4006 10.5526L22.3179 7.48227L22.3217 7.48047ZM21.7293 6.39847C21.7293 6.60587 21.6714 6.81517 21.5536 7.00017L11.6084 22.9178C11.5187 23.0617 11.3748 23.0785 11.3169 23.0785C11.2589 23.0785 11.1169 23.0617 11.0254 22.9178L8.51195 18.8908C8.28396 18.5264 8.28396 18.0573 8.51195 17.6892L18.4572 1.77527C18.5469 1.62951 18.6908 1.6127 18.7487 1.6127C18.8066 1.6127 18.9505 1.62951 19.0421 1.77714L21.5536 5.79677C21.6714 5.98177 21.7293 6.19097 21.7293 6.39847Z" fill="#1D1F20"/>
<path d="M29.7538 17.2063L27.2422 13.1867C27.2422 13.1867 27.2329 13.1699 27.2273 13.1624L27.0292 12.8466C26.6555 12.2468 26.0089 11.888 25.3025 11.8861L21.2568 11.873L20.9746 12.3253L16.1403 20.0617L15.873 20.4896L17.8987 23.7262C18.2725 24.328 18.919 24.6867 19.631 24.6867H25.3007C25.9996 24.6867 26.6611 24.3186 27.0311 23.7281L27.231 23.4086C27.231 23.4086 27.2385 23.3992 27.2404 23.3955L29.7556 19.3703C30.1686 18.7125 30.1686 17.8622 29.7556 17.2063H29.7538ZM28.9876 18.89L26.4723 22.9152C26.4611 22.9339 26.448 22.9488 26.4368 22.9638C26.349 23.0628 26.235 23.0759 26.1846 23.0759C26.1266 23.0759 25.9846 23.0591 25.893 22.9133L23.3778 18.8882C23.3497 18.8433 23.3254 18.7966 23.303 18.7461C23.2806 18.6975 23.2638 18.649 23.2488 18.5985C23.1928 18.3967 23.1928 18.1799 23.2488 17.9781C23.2769 17.8791 23.3198 17.78 23.3759 17.6903L25.8874 13.6689C25.8874 13.6689 25.8912 13.6633 25.893 13.6595C25.9528 13.5698 26.0276 13.5287 26.093 13.5156C26.1192 13.5082 26.1416 13.5063 26.1603 13.5026C26.1696 13.5026 26.1789 13.5026 26.1883 13.5026C26.2462 13.5026 26.3901 13.5212 26.4798 13.667L28.9913 17.6866C29.2212 18.0528 29.2212 18.5238 28.9913 18.89H28.9876Z" fill="#1D1F20"/>
</svg>

Em código

Comment (mateusmenesesDev) [reply public/favicon.svg:1]: `<svg width="31" height="25" viewBox="0 0 31 25" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M22.3217 7.48047C22.7328 6.82077 22.7328 5.97237 22.3217 5.31647L19.8101 1.29689L19.6008 0.95865C19.2252 0.3588 18.5787 0 17.8685 0H12.1989C11.4907 0 10.8441 0.3588 10.4666 0.96052L0.313942 17.2089C0.110254 17.5341 0 17.9097 0 18.2909C0 18.6721 0.108385 19.0477 0.312074 19.371L3.03478 23.7307C3.41038 24.3324 4.05696 24.6893 4.76519 24.6893H10.4348C11.1468 24.6893 11.7934 24.3306 12.1671 23.7288L12.3746 23.3999C12.3746 23.3999 12.3746 23.3999 12.3746 23.3962L12.3783 23.3887L14.4021 20.1521L20.4006 10.5526L22.3179 7.48227L22.3217 7.48047ZM21.7293 6.39847C21.7293 6.60587 21.6714 6.81517 21.5536 7.00017L11.6084 22.9178C11.5187 23.0617 11.3748 23.0785 11.3169 23.0785C11.2589 23.0785 11.1169 23.0617 11.0254 22.9178L8.51195 18.8908C8.28396 18.5264 8.28396 18.0573 8.51195 17.6892L18.4572 1.77527C18.5469 1.62951 18.6908 1.6127 18.7487 1.6127C18.8066 1.6127 18.9505 1.62951 19.0421 1.77714L21.5536 5.79677C21.6714 5.98177 21.7293 6.19097 21.7293 6.39847Z" fill="#1D1F20"/>
<path d="M29.7538 17.2063L27.2422 13.1867C27.2422 13.1867 27.2329 13.1699 27.2273 13.1624L27.0292 12.8466C26.6555 12.2468 26.0089 11.888 25.3025 11.8861L21.2568 11.873L20.9746 12.3253L16.1403 20.0617L15.873 20.4896L17.8987 23.7262C18.2725 24.328 18.919 24.6867 19.631 24.6867H25.3007C25.9996 24.6867 26.6611 24.3186 27.0311 23.7281L27.231 23.4086C27.231 23.4086 27.2385 23.3992 27.2404 23.3955L29.7556 19.3703C30.1686 18.7125 30.1686 17.8622 29.7556 17.2063H29.7538ZM28.9876 18.89L26.4723 22.9152C26.4611 22.9339 26.448 22.9488 26.4368 22.9638C26.349 23.0628 26.235 23.0759 26.1846 23.0759C26.1266 23.0759 25.9846 23.0591 25.893 22.9133L23.3778 18.8882C23.3497 18.8433 23.3254 18.7966 23.303 18.7461C23.2806 18.6975 23.2638 18.649 23.2488 18.5985C23.1928 18.3967 23.1928 18.1799 23.2488 17.9781C23.2769 17.8791 23.3198 17.78 23.3759 17.6903L25.8874 13.6689C25.8874 13.6689 25.8912 13.6633 25.893 13.6595C25.9528 13.5698 26.0276 13.5287 26.093 13.5156C26.1192 13.5082 26.1416 13.5063 26.1603 13.5026C26.1696 13.5026 26.1789 13.5026 26.1883 13.5026C26.2462 13.5026 26.3901 13.5212 26.4798 13.667L28.9913 17.6866C29.2212 18.0528 29.2212 18.5238 28.9913 18.89H28.9876Z" fill="#1D1F20"/>
</svg>
`

**Review by mateusmenesesDev — APPROVED** [review]

**Pull request merged**